src/generate_mhq_sample/20_mhq_terr_sample_admin.Rmd

```diff
@@ -0,0 +1,1011 @@
+# Generate mhq sample administration table
```

Comment:
General remark about the subparagraphs in this part (e.g. below 'Samples assigned to INBO'): while the textual explanations are reasonable, it is still difficult to follow what is actually done and what the outcome is, if the resulting (intermediate) objects are not somehow visualised (e.g. with `glimpse()`, first rows of a table etc.). While such visualisation is done to enlist anomalies, it isn't for the main data workflow. This makes it uneasy to understand what is meant with titles like '_Combine sample table with admin data_', '_Combine sample admin data and assessments data_' etc. without running code in an interactive session and inspecting results. In such cases the accompanying text focuses on specific aspects, not on explanation of the actual step (it's my impression, maybe partly wrong).